docs-archive/1.10.4/changelog.html

```diff
@@ -620,7 +620,7 @@ <h3>Doc-only changes<a class="headerlink" href="#doc-only-changes" title="Permal
 <li><p>[AIRFLOW-XXX] Reduce log spam in tests (#5174)</p></li>
 <li><p>[AIRFLOW-XXX] Speed up tests for PythonSensor (#5158)</p></li>
 <li><p>[AIRFLOW-XXX] Add Bas Harenslak to committer list (#5157)</p></li>
-<li><p>[AIRFLOW-XXX] Add Jarek Potiuk to commiter list (#5132)</p></li>
+<li><p>[AIRFLOW-XXX] Add Jarek Potiuk to committer list (#5132)</p></li>
```

Comment:
This change should be made to the `apache/airflow` repo too (and before this PR is merged.)

Reply:
ah yeah. Thanks for pointing this out. I opened the PR here https://github.com/apache/airflow/pull/7392

I must admit I haven't looked into the docs part of the website yet.